File: dotnet/Lexy/Lexy.Compiler/Compiler/CSharp/BuiltInFunctions/SecondsFunctionCall.cs
using Lexy.Compiler.Language.Expressions.Functions;
using Lexy.RunTime;

namespace Lexy.Compiler.Compiler.CSharp.BuiltInFunctions;

internal class SecondsFunctionCall : EndStartDateFunctionCall<SecondsFunction>
{
    protected override string ClassName => nameof(BuiltInDateFunctions);
    protected override string MethodName => nameof(BuiltInDateFunctions.Seconds);
}

File: typescipt/src/compiler/JavaScript/builtInFunctions/secondsFunctionCall.ts
import {EndStartDateFunctionCall} from "./endStartDateFunctionCall";
import {SecondsFunction} from "../../../language/expressions/functions/secondsFunction";

export class SecondsFunctionCall extends EndStartDateFunctionCall {

   protected override className = "BuiltInDateFunctions";
   protected override methodName = "seconds";

   constructor(functionNode: SecondsFunction) {
      super(functionNode);
   }
}
